docs/pipelineruns.md

```diff
@@ -163,12 +163,29 @@ spec:
 
 You may also use a `Tekton Bundle` to reference a pipeline defined remotely.
 
+{{< tabs >}}
+{{< tab "v1" >}}
+spec:
+  pipelineRef:
+    resolver: bundles
```

Comment:
This syntax is also supported in v1beta1-- it's probably worth mentioning that and also linking to the resolver docs. (Here and in the other bundle example)